Philippines/salamat.md

```diff
@@ -1,3 +1,5 @@
 Maligayang pasko at manigong bagong taon sa inyo! Salamat sa Udacity para sa
 
-online class na ito :D
\ No newline at end of file
+online class na ito :D
+
+Para matuto, gumawa ka ng links tulad [nito](Philippines/bagongpanimula.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.